### PR TITLE
feat: move cds annotation converter package

### PR DESCRIPTION
**Pull request by nahmed22**
Move cds annotation converter module to open-ux-tools repository

### CODE REVIEW

**Review by vinayhospete — DISMISSED** [review]

**Comment by vinayhospete** [comment]
Please solve 6 sonar issues (remove TODO comments in code)

**Review by vinayhospete — DISMISSED** [review]

**Review by voicis — DISMISSED** [review]

File: .changeset/pretty-coins-leave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@sap-ux/cds-odata-annotation-converter": patch
```
Comment (kranthie-sap) [inline .changeset/pretty-coins-leave.md:2]: Consider changing this to a minor upgrade.

**Review by kranthie-sap — CHANGES_REQUESTED** [review]
The changes to the codeowners file, sonar configuration and root tsconfig.json look good. Please see comment about changeset file.

File: .changeset/pretty-coins-leave.md (hunk 1 above)
Comment (kranthie-sap) [reply .changeset/pretty-coins-leave.md]: Thanks, @voicis

**Review by kranthie-sap — APPROVED** [review]

**Review by voicis — APPROVED** [review]

**Pull request merged**